payments/voucher_manager.go

```diff
@@ -5,6 +5,7 @@ import (
 	"math/big"
 
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/statechannels/go-nitro/client/engine/store/safesync"
```

Comment:
Probably beyond the scope of this PR but the `safesync` package could probably live outside the `store` now that it's being used beyond the `store` package.

That might prevent some annoying import cycles in the future.